ai/voice completion creator/68ca127a-2a6a-46c4-aaea-3651cd5511e9.json

```diff
@@ -185,10 +185,13 @@
     "ask": null,
     "question": null,
     "answer": null,
-    "openAI": null
+    "openAI": null,
+    "azure": null,
```

Comment:
Provider names like Azure and OpenAI was not added to the other prototypes as keyword.
I think we should either remove them here or add them to the others too.

Reply:
This is a complex strucutre node which uses both OpenAI and azure services, but it is under the ai namespace. Adding the keyword to the othersis unneccesary, because they are listed in the search due to their namespaces.

Reply:
Indeed.